lib/board.rb

```diff
@@ -0,0 +1,51 @@
+class Board
+  attr_accessor :cells
+
+  def initialize
+    self.reset!
+  end
+
+  def reset!
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(input)
+    cells[input.to_i - 1]
+  end
+
+  def update(pos, player)
+    cells[pos.to_i - 1] = player.token
+  end
+
+  def full?
+   cells.detect {|cell| cell == " " || cell == nil} ? false : true
```

Comment:
1. Prefer implicit return values of boolean statements.

2. A ternary operator that returns `true` or `false` is never needed.

3. Use `!` negation to reverse the "truthyness" of the boolean. `!cells.detect {|cell| cell == " " || cell == nil}`